docs/validations-action-repo.md

```diff
@@ -0,0 +1,116 @@
+## Validations for an Action repository
```

Comment:
Should have this and automated tests as sub sections in the TOC in the readme?

Reply:
'Automated tests' is already there, will add this as well.